scripts/mcp-start-mock.sh

```
@@ -0,0 +1,9 @@
+#!/bin/bash
+# Wrapper script for MCP Inspector to run the TypeScript CLI in mock mode
+cd "$(dirname "$0")/.."
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)